.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 out
 .vscode/settings.json
+*.tgz
```

Comment:
Missing a newline at the end of the file.

I'm guessing this change is for `npm pack` testing?

Reply:
Yes, this was for my testing purposes as I wanted to ensure the plugin worked. I can leave it out if preferred.